packages/backend/src/graphql/schema.graphql

```diff
@@ -400,6 +405,15 @@ type PaymentPointer implements Model {
   ): OutgoingPaymentConnection
   "Date-time of creation"
   createdAt: String!
+  "Status of the payment pointer"
+  status: PaymentPointerStatus
```

Comment:
If we get rid of `status` in the model, this could potentially just be `isActive` with a boolean. No need to define another enum then.

Reply:
This was my suggestion, I typically would prefer an enum over a boolean for APIs like this

Reply:
This is the root of having both `isActive` and `status`.

Whatever abstractions we want for the `deactivatesAt` field, I think it needs to be on the model. So if we return a status in gql then I think it needs to be on the model. Otherwise we would have to derive the status from either `deactivatesAt` (or isActive if exists) everywhere we are returning a payment pointer in gql (create, update, get, and maybe other resolvers).

So I think the fundamental question is what we want to return in the API. My initial assumptions was a boolean because that's all that was necessary. Max suggested an enum and I was receptive to points about it being more readable/explicit. Most importantly I want to be consistent with our other responses. Less importantly, I think it could handle (possibly very unlikely) situations like adding another status like "PENDING".

For these reasons I was actually leaning towards ditching `isActive` for `status`, if anything.

Reply:
> So if we return a status in gql then I think it needs to be on the model. Otherwise we would have to derive the status from either deactivatesAt (or isActive if exists) everywhere we are returning a payment pointer in gql (create, update, get, and maybe other resolvers).

Thinking about this more, I guess I could add a status resolver on our PaymentPointer resolver here:
```
  PaymentPointer: {
    incomingPayments: getPaymentPointerIncomingPayments,
    outgoingPayments: getPaymentPointerOutgoingPayments,
    quotes: getPaymentPointerQuotes
  }
  ```
If we want to keep returning the status on the API but use `isActive` on the backend.

Reply:
> Will go with isActive if we prefer that but I do think it's tied to what we want to return in the API.

i.e. we keep everything as is, with `status` in the GraphQL API and `isActive` as a "helper" method in the application layer model?

Reply:
I removed status from the model and kept `isActive`. I left status in the graphql layer as a field on paymentPointer and an option in the update input.